configure.ac

```diff
@@ -2198,6 +2198,10 @@ AC_CHECK_DECL([CLOCK_MONOTONIC],
 	 AC_DEFINE([HAVE_CLOCK_MONOTONIC], [1], [Have monotonic clock])
 ], [AC_MSG_RESULT([no])], [FRR_INCLUDES])
 
+AC_CHECK_DECL([CLOCK_THREAD_CPUTIME_ID], [
```

Comment:
please add this change to doc/user/installation.rst

Reply:
Uh ... it's not an option to switch?  ... it does the same thing as `getrusage()` ... it's used if it's available ... and has been around forever ... I'm honestly not sure what to put in the docs about this?

Reply:
ignore me my brain is mush lately